package.json

```diff
@@ -25,6 +25,7 @@
     "html-loader": "^0.2.3",
     "imports-loader": "^0.6.3",
     "jquery.cookie": "^1.4.1",
+    "jshint": "^2.8.0",
```

Comment:
This should be added in `devDependencies` so that it doesn't get installed on server deployments.